lm_tournament_eval/api/model_utils.py

```diff
@@ -34,14 +124,28 @@ def load_model(model_type, model, model_args, batch_size: int = 1,
             logging.info(
                 f"Initializing {model} model, with arguments: {simple_parse_args_string(model_args)}"
             )
-            lm = get_model(model_type).create_from_arg_string(
-                model_args,
-                {
-                    "batch_size": batch_size,
-                    "max_batch_size": max_batch_size,
-                    "device": device,
-                }
-            )
+
+            match model_type:
+                case "hf":
+                    lm = get_model(model_type).create_from_arg_string(
+                        model_args,
+                        {
+                            "batch_size": batch_size,
+                            "max_batch_size": max_batch_size,
+                            "device": device,
+                        }
+                    )
+                case "anthropic" | "anthropic-chat" | "anthropic-chat-completions":
```

Comment:
is there a reason other than being succinct these are separate cases for anthropic/openai? it looks like the code following is the same

Reply:
I think we should keep the cases separate because I think that the APIs may support slightly different arguments that we might eventually want to pass in those currently empty dictionaries - if that's true we would want to do different validation actions in each case.